contrib/lang/ruby/config.el

```diff
@@ -15,3 +15,5 @@
 (defvar ruby-version-manager nil
   "If non nil defines the Ruby version manager (i.e. rbenv, rvm)")
 
+(defvar ruby-on-rails-support nil
```

Comment:
to be consistent with the other layer variables, consider renaming it to `ruby-enable-ruby-on-rails-support`